src/coreclr/nativeaot/System.Private.CoreLib/src/System.Private.CoreLib.csproj

```diff
@@ -257,6 +257,12 @@
   <ItemGroup Condition="'$(TargetsWindows)'=='true'">
     <Compile Include="Microsoft\Win32\SafeHandles\SafeThreadPoolIOHandle.cs" />
     <Compile Include="System\Runtime\InteropServices\PInvokeMarshal.Windows.cs" />
+    <Compile Include="$(CommonPath)\System\Runtime\InteropServices\IDispatch.cs">
```

Comment:
This should not be necessary once your dotnet/runtime change gets in. Is that rigth? It may be better to cherry pick that change into this PR and delete this line.